Rakefile

```diff
@@ -4,3 +4,75 @@ require "rspec/core/rake_task"
 RSpec::Core::RakeTask.new(:spec)
 
 task :default => :spec
+
+namespace :openapi do
+  IBM_CLOUD_GEMS = {
+    "ibm_cloud_iam"                 => "iam-identity-token-api.json",
+    "ibm_cloud_power"               => "power-cloud.json",
+    "ibm_cloud_resource_controller" => "resource-controller/resource-controller.json",
+  }
+
+  task :download_cli, [:version] do |_t, args|
+    version = args[:version] || "5.0.0-beta2"
+
+    output_file    = "openapi-generator-cli-#{version}.jar"
+    target_symlink = "openapi-generator-cli"
+
+    unless File.exist?(output_file)
+      puts "Downloading #{output_file}..."
+
+      uri = URI("https://repo1.maven.org/maven2/org/openapitools/openapi-generator-cli/#{version}/#{output_file}")
+
+      open(output_file, "wb") { |f| download(uri, f) }
+
+      puts "Downloading #{output_file}...Complete"
+    end
+
+    File.unlink(target_symlink) if File.exist?(target_symlink)
+    File.symlink(output_file, target_symlink)
+  end
+
+  task :download_openapi_specs do
+    require "json"
```

Comment:
Super minor, but I don't think this line it needed.

Reply:
:+1: you're right, I was playing with auto-fixing the vpc spec before generating the clients but now we're deferring that

Reply:
Removed